test/operator-framework-e2e/operator_framework_test.go

```diff
@@ -6,125 +6,86 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-
 	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/rand"
-	"k8s.io/client-go/rest"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	catalogd "github.com/operator-framework/catalogd/api/core/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	operatorv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
 )
 
-var (
-	cfg *rest.Config
-	c   client.Client
-	ctx context.Context
-)
-
 func TestOperatorFramework(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Operator Framework E2E Suite")
-}
+	t.Parallel()
+	cfg := ctrl.GetConfigOrDie()
 
-var _ = BeforeSuite(func() {
-	cfg = ctrl.GetConfigOrDie()
 	scheme := runtime.NewScheme()
-	ctx = context.Background()
 
-	err := catalogd.AddToScheme(scheme)
-	Expect(err).ToNot(HaveOccurred())
+	utilruntime.Must(catalogd.AddToScheme(scheme))
```

Comment:
If you have a `t *testing.T` present, you might as well use `t.Fatal()` - but, why are we creating so many Schemes? There could just as well be a package somewhere in this repo that exposes a global.

Reply:
We have #322. I think this is what you are suggesting here.

Reply:
Yes, this has a `t` so, I can use the that. But since there's an issue for schemes already, we can use that to update the schemes.